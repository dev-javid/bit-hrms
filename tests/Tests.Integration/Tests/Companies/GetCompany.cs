namespace Tests.Integration.Tests.Companies
{
    public class GetCompany(TestWebApplicationFactory<Program> factory) : IntegrationTest(factory)
    {
        private const string Route = "/api/companies/999";

        [Fact]
        [UseReporter(typeof(DiffReporter))]
        public async Task Get_Company()
        {
            await LoginAsSuperAdminAsync();
            using HttpResponseMessage response = await Client.GetAsync(Route);
            response.StatusCode.Should().Be(HttpStatusCode.OK);
            var result = await response.Content.ReadAsStringAsync();
            Approvals.VerifyJson(result);
        }
    }
}
