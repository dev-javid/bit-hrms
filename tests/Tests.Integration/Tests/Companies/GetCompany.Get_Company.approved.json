{
    "weeklyOffDays": [
        "Sunday"
    ],
    "companyId": 999,
    "name": "Test Company",
    "phoneNumber": "9876543210",
    "financialMonth": 3,
    "administratorName": "Super Admin",
    "email": "company1@example.com",
    "address": "Test Address"
}